src/core/mac/mac_frame.cpp

```diff
@@ -529,6 +641,9 @@ bool Frame::IsSrcPanIdPresent(uint16_t aFcf)
         present = false;
     }
 
+#if OPENTHREAD_CONFIG_WAKE_UP_COORDINATOR_ENABLE || OPENTHREAD_CONFIG_WAKE_UP_END_DEVICE_ENABLE
```

Comment:
Same here about adding `ExitNow()` somewhere to avoid the `#if` around `exit` label?